pkg/webhook/validation/validator.go

```diff
@@ -51,3 +56,14 @@ func (v *validator) validate(policy *v1alpha1.AuthorizationPolicy) error {
 	}
 	return nil
 }
+
+func (v *validator) validateVpol(policy *v1alpha1.ValidatingPolicy) error {
+	if allErrs := v.compileVpol(policy); len(allErrs) > 0 {
```

Comment:
You should check that evaluation mode is `Envoy` here, there's no reason to do any kind of validation if the policy doesn't belong to the envoy plugin.